v2/openapi/fb-provider-collateral-api.yaml

```diff
@@ -257,7 +257,49 @@ paths:
           $ref: '#/x-responses/ErrorResponse'
         '5XX':
           $ref: '#/x-responses/ErrorResponse'
-
+  /accounts/{accountId}/collateral/{collateralId}/intents/deposits:
+    post:
+      tags: [ collateral ]
+      summary: Initiate a collateral deposit transaction
+      description: >
+        Notifies the provider to have start listening to a new collateral deposit transaction.
+        The provider is expected to listen to this address and credit the account accordingly
+      operationId: initiateCollateralDepositTransaction
+      parameters:
+        - $ref: '#/x-header-params/X-FB-PLATFORM-SIGNATURE'
+        - $ref: '#/x-header-params/X-FBAPI-KEY'
+        - $ref: '#/x-header-params/X-FBAPI-NONCE'
+        - $ref: '#/x-header-params/X-FBAPI-SIGNATURE'
+        - $ref: '#/x-header-params/X-FBAPI-TIMESTAMP'
+        - $ref: '#/x-path-params/SubAccountIdPathParam'
+        - $ref: '#/components/parameters/CollateralIdPathParam'
+      requestBody:
+        description: Collateral deposit transaction details
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/CollateralDepositTransactionIntentRequest'
+      responses:
+        '200':
+          description: Successful Operation
+          headers:
+            X-PROVIDER-SIGNATURE:
+              $ref: '#/x-headers/X-PROVIDER-SIGNATURE'
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/CollateralDepositTransactionIntentResponse'
```

Comment:
We said we're currently not going to have an intent response per endpoint family